internal/warehouse/github_actions_test.go

```diff
@@ -0,0 +1,443 @@
+package warehouse
+
+import (
+	"context"
+	"net/http"
+	"net/url"
+	"os"
+	"testing"
+
+	"github.com/golang/mock/gomock"
+	"github.com/google/go-github/v47/github"
+	"github.com/google/uuid"
+	"github.com/jackc/pgx/v4"
+	"github.com/mergestat/mergestat/internal/db"
+	"github.com/mergestat/mergestat/internal/helper"
+	"github.com/mergestat/mergestat/internal/mocks"
+	"github.com/migueleliasweb/go-github-mock/src/mock"
+	"github.com/rs/zerolog"
+)
+
+func Test_warehouse_handleWorkflows(t *testing.T) {
```

Comment:
Let's follow the Go conventions here on all these test names please: https://go.dev/doc/effective_go#mixed-caps

Reply:
nice catch , I forgot about this one :)